src/containers/BooksForm.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createID } from '../initState';
+import { createBook } from '../actions/index';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: createID(),
+      title: '',
+      category: categories[0],
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange({ target }) {
```

Comment:
- [ ] [OPTIONAL] If you declare `handleChange` and `handleSubmit` using arrow functions instead of the `function` reserved word you won't need to to the binding inside the constructor and will save 2 lines of code! 
[Read more here. ](https://medium.com/@joespinelli_6190/using-arrow-functions-to-avoid-binding-this-in-react-5d7402eec64)